sos/report/plugins/shmcli.py

```diff
@@ -0,0 +1,109 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, IndependentPlugin, PluginOpt
+
+
+class SHMcli(Plugin, IndependentPlugin):
+    """shmcli pulls hardware information from PowerVault/Dell
+    Storage JBOD's attached to server.
+    It provides information of the adapters, emms, drives,
+    enclosures, fans, power supplies and the sensory data of
+    temp, voltage, and current sensors.
+    """
+
+    short_desc = 'Dell Server Hardware Manager'
+
+    plugin_name = 'shmcli'
+    profiles = ('system', 'storage', 'hardware',)
+
+    files = ('/opt/dell/ServerHardwareManager/' +
+             'ServerHardwareManagerCLI/bin/shmcli',)
+
+    option_list = [
+        PluginOpt('debug', default=False, desc='capture support debug data')
+    ]
+
+    def setup(self):
+        cmd = "/opt/dell/ServerHardwareManager/" \
+              "ServerHardwareManagerCLI/bin/shmcli"
```

Comment:
It may have a sense to have a `shmcli_bin` (or directly `cmd`) variable common for the class, and refer `files = (cmd,)` to de-duplicate the long filepath. But never mind, this is also OK.